### DISCUSSION TITLE
-cc msc missing

### DISCUSSION THREAD

**Discussion by Jarod42** [premake/premake-core#1998] [General]
Is there any reason to restrict --cc option of premake to gcc/clang and not msc?

Not all generators support msc (ok, from core, only codelite claims to support it, I though gmake* support it too, but not) (as not all generator support gcc/clang).

**Comment by Jarod42** [comment]
So added https://github.com/premake/premake-core/pull/2003 :)

**Reply by samsinsane** [reply]
I'm going to close this discussion based on this PR being merged.

**Comment by samsinsane** [comment]
I doubt it was _restricted_ to those values, I would say that someone wanted to specifically set GCC or Clang and that's just what the option had.

> I though gmake* support it too, but not

Maybe one day 🤞
